app/v1/models/response/submit/SubmitResponse.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.response.submit
+
+import play.api.libs.json.{Json, OWrites, Reads}
+
+case class SubmitResponse(processingDate: String,
```

Comment:
Its not in the spec, but something I remembered about old VAT - the processing date might not always be returned from DES with milliseconds, but we want to add them on if they are not there.  Take a look at the old VatReturnsDES model to see what we did last time (there might be a better way)

Reply:
The test for the original code is here https://github.com/hmrc/vat-api/blob/master/func/uk/gov/hmrc/vatapi/resources/ValueAddedTaxReturnsRetrievalSpec.scala#L107-L123
Definitely don't copy that test format, just for an DYI